.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
```

Comment:
this is fine for now, but I would prefer pinning to digest (sha sum). I only know how to do this for rennovate:

https://github.com/numba/llvmlite/pull/1156/files

if possible can we do this for dependabot too?

Reply:
Also, rennovate can update the pre-commit config.

Reply:
The version above is for dependabot's manifest.  Not sure if it can pin version using hashes.  We can just switch to renovate if you think it is a better tool.

Reply:
https://github.com/wiki2beamer/wiki2beamer/issues/49